docs/machine_learning.rst

```diff
@@ -89,30 +89,42 @@ Finally execute this whole training flow as a batch job::
     training_job = model.create_job()
     training_job.start_and_wait()
 
-
 Inference
 ----------
 
 When the batch job finishes successfully, the trained model can then be used
 with the ``predict_random_forest`` process on the raster data cube
 (or another cube with the same band structure) to classify all the pixels.
 
+We inspect the result metadata of the training job to obtain the STAC Item URL of the trained model::
+
+
+    results = training_job.get_results()
+    links = results.get_metadata()['links']
+    ml_model_metadata_url = [link for link in links if 'ml_model_metadata.json' in link['href']][0]['href']
+    print(ml_model_metadata_url)
+
+
+Next, load the model from the URL::
+
+    model = connection.load_ml_model(id=ml_model_metadata_url)
```

Comment:
just do
```suggestion
    model = connection.load_ml_model(ml_model_metadata_url)
```

I think the `id` argument is not a very good name and we might change that in the future, so just avoid using it to keep these docs more robust

Reply:
ref:
- https://github.com/Open-EO/openeo-python-driver/issues/306